srecord/record.cc

```diff
@@ -28,9 +28,10 @@ srecord::record::record(const srecord::record &arg) :
     address(arg.address),
     length(arg.length)
 {
-    if (arg.length > 0)
+    if (arg.length > 0) {
```

Comment:
Coding style is to have no braces around single statement conditional. If braces are used, the opening brace is on the next line aligned with the start of the 'if' keyword
```suggestion
    if (arg.length > 0) // and delete the closing brace
```